### PR TITLE
Rearrange array in alternating positive & negative items 

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes 

### Description

using the single-pass approach 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue is related #5847

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Not getting what your finding in output you wrote intersection and in description something else.

**Comment by soumikbaithalu** [comment]
> Not getting what your finding in output you wrote intersection and in description something else.

I don't understand my code is working and print correct OUTPUT  as the question said

**Review by ankitaggarwal23 — COMMENTED** [review]
Capture

What you mention rearranging array and what you are finding common elements

**Comment by soumikbaithalu** [comment]
@ankitaggarwal23 please check my modify file

File: Python/other/Rearrange_the_neagtive_positive_element.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+If there are more positive numbers they appear at the end of the array.
+If there are more negative numbers, they too appear in the end of the array.
+
+Examples : 
+
+Input:  arr[] = {1, 2, 3, -4, -1, 4}
+Output: arr[] = {-4, 1, -1, 2, 3, 4}
+
+Input:  arr[] = {-5, -2, 5, 2, 4, 7, 1, 8, 0, -8}
+output: arr[] = {-5, 5, -2, 2, -8, 4, 7, 1, 8, 0}
```
Comment (ankitaggarwal23) [inline Python/other/Rearrange_the_neagtive_positive_element.py]: Remove this part as already you wrote output

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Rest LGTM.

File: Python/other/Rearrange_the_neagtive_positive_element.py (hunk 1 above)
Comment (soumikbaithalu) [reply Python/other/Rearrange_the_neagtive_positive_element.py]: Done 👍🏻 @aakriti1318

**Comment by HarshCasper** [comment]
Please resolve the merge conflicts.

**Comment by soumikbaithalu** [comment]
Screenshot (217)

i can't resolve the file see what is showing

**Comment by soumikbaithalu** [comment]
> Please resolve the merge conflicts.

@HarshCasper it is resolved by your side I can't do any thing from my side

**Pull request closed without merging**